frontend/pages/_app.tsx

```diff
@@ -62,7 +63,9 @@ const App = ({ Component, pageProps }: AppProps) => {
         />
         <UserInfoProvider>
           <Main className={inter.className}>
-            <Component {...pageProps} />
+            <ErrorBoundary>
```

Comment:
Could it make sense to move this up a few levels or should it be next to the component? `UserInfoProvider` could also error I guess?

Or would that not work?

Reply:
Nevermind, I just started reading up on it